### PR TITLE
feat(evmlib): Chunk->Data

### PR DESCRIPTION
**Pull request by joshuef**

### CODE REVIEW

File: sn_evm/CHANGELOG.md [hunk 1]
```diff
@@ -266,7 +266,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [0.14.35](https://github.com/maidsafe/safe_network/compare/sn_transfers-v0.14.34...sn_transfers-v0.14.35) - 2024-01-09
 
 ### Added
-- *(client)* extra sleep between chunk verification
+- *(client)* extra sleep between data verification
```
Comment (grumbach) [inline sn_evm/CHANGELOG.md:270]: Not sure we should change this in the Changelog file too!!

Comment (joshuef) [reply sn_evm/CHANGELOG.md:270]: ups! FNR gone TOO FAR

File: evmlib/artifacts/DataPayments.json
Comment (mickvandijke) [inline evmlib/artifacts/DataPayments.json:1]: I need to update the smart contract first to reflect these name changes in here. 

Can this be left out in this PR? I'll open up a follow-up PR with the updated contract and new contract address.

Comment (joshuef) [reply evmlib/artifacts/DataPayments.json:1]: im not sure what parts need left out (but yeh, whatever makes sense)

Comment (joshuef) [reply evmlib/artifacts/DataPayments.json:1]: (is the contract not in here? I thought it was contained in the evm crate)

**Pull request closed without merging**